src/server/saved_objects/client/lib/create_find_query.js

```diff
@@ -32,19 +45,27 @@ export function createFindQuery(mappings, options = {}) {
   }
 
   if (search) {
+    let searchFields = options.searchFields;
+    // If options.searchFields is a singular string field, turn it into an array of one.
+    searchFields = searchFields && !Array.isArray(searchFields) ? [searchFields] : searchFields;
```

Comment:
no need to update, but you can do `[].concat(searchFields)` if you want to be fancy.  imo it's moderately easier to parse through if it's an array all the way through